main.tex

```diff
@@ -904,70 +934,55 @@
 
-\item  the theft of any vehicle or of any item within any vehicle parked in the car park;
+\item  the theft of any vehicle or of any item within any vehicle parked in the garage;
 
-\item  vehicles are parked and left in the car park as this is at the sole risk of the Lot Owner or Occupier.
+\item  vehicles are parked and left in the garage as this is at the sole risk of the Lot Owner or Occupier.
 
 \end{enumerate}
 
-\item  An Owner or Occupier of a Lot may only wash and clean their vehicles in the designated car wash bay (if applicable) and must seek approval of the Manager should they require the use of the car wash bay for in excess of one hour.
+\item  An Owner or Occupier of a Lot may only wash and clean their vehicles in the designated vehicle-washing bay (if applicable) and must seek approval of the Manager or Owners Corporation should they require the use of the vehicle-washing bay for in excess of one hour.
 
 \end{enumerate}
 
 \item  Central Courtyard Without limiting any other provision of these Rules, the following rules apply to the use of the central courtyard which forms part of the Common Property:
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  The initial opening hours for the central courtyard will be 7am to 10pm. The Manager reserves the right to modify the opening hours for the central courtyard from time to time by way of a written notice to Owners and Occupiers or by placing visible signage of the amended opening hours in or about the central courtyard or the Development.
+\item  The initial opening hours for the central courtyard will be 7am to 10pm. The Owners Corporation reserves the right to modify the opening hours for the central courtyard from time to time by way of a written notice to Owners and Occupiers or by placing visible signage of the amended opening hours in or about the central courtyard or the Development.
 \newpage
 
 
-
-\item  Owners and Occupiers must comply with all reasonable:
-
-\begin{enumerate}[label=(\alph*)]
-
-\item  rules in respect of use of the central courtyard as determined by the Manager from time to time; and
-
-\item  directions given by the Manager in connection with use of the central courtyard.
-
-\end{enumerate}
-
-\item  If a noise or behavioural complaint is made against any Owner or Occupier in respect of that Owner’s or Occupier’s use or occupation of the central courtyard, the Manager may direct that the Owner or Occupier cease the noise or activity, or leave the central courtyard, and the Owner or Occupier must immediately comply with that direction.
-
-\item  Only background music shall be played within the central courtyard, and for this purpose, a mini portable speaker can be brought into the central courtyard. For the purpose of this Rule, ‘background music’ means music at a level that enables Owners, Occupiers or invitees to conduct an uninterrupted or undisturbed conversation at a distance of 600mm within the central courtyard
```

Comment:
This has already been covered by the section about Reasonable Use of Common Property